tools/sync_test_server/setup_mongodb_realm.sh

```diff
@@ -42,7 +42,7 @@ yes | stitch-cli login --config-path=/tmp/stitch-config \
 IMPORT_RESPONSE=$(stitch-cli import \
                   --config-path=/tmp/stitch-config \
                   --base-url=http://localhost:9090 \
-                  --path=/tmp/app_config \
+                  --path=/tmp/app_config_empty \
```

Comment:
I actually think the above is a bit cleaner...at least conceptually. But as it did not fix the issue I reverted the changes to avoid cluttering with these extra config files for now. 

The docker image with the server already includes a `run.sh` handling importing of apps, so we could basically get the same import as I did by just mounting our app in the `/apps`-directory in the image. Only issues are that:
- It is harder to track when apps have been fully imported (but can be tailed by `docker logs --follow mongodb-realm` or something like that)
- I do not know how I should get the actual _app id_ to the endpoint where we fetch it from the device test.

I think fixing the above would be the right way to go, as we would not have to maintain our own script and we would avoid having two imports running in parallel. Cannot see the immediate explanation for this to trigger #6904, but I guess anything that could simplify setup would be worth doing.

Reply:
If the script just stores the app id in a text file we should be able to read it for our usage I think